data/hai/unfettered side missions.txt

```diff
@@ -95,7 +95,11 @@ mission "Hai Legacy 1"
 					decline
 			label first
 			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull on your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ships, as if to mimic the actions of a regular passerby, they address you again.`
-			`	"My associates and I have a lucrative business proposition for you. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location, and if you cross-reference it with the local maps, it appears to point towards a beach on the other side of the planet.`
+			`	"My associates and I have a lucrative business proposition for you. It will require your undivided attention, so you would do well to finish any ongoing business first. If you are interested, come find us at these coordinates."`
+			`	"We note you operate multiple ships. However, this is a solitary affair; make sure you can park most of your ships."`
```

Comment:
This second line only display if you have more than one ship, but if it does display then the previous line shouldn't end in its quotation mark. Instead of adding another `to display`, you should start this second line with something that isn't a quotation so that the previous line is correct regardless of which line comes next.